src/Gui/Inventor/Draggers/Gizmo.cpp

```diff
@@ -0,0 +1,697 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
+/****************************************************************************
+ *                                                                          *
+ *   Copyright (c) 2025 Sayantan Deb <sayantandebin[at]gmail.com>           *
+ *                                                                          *
+ *   This file is part of FreeCAD.                                          *
+ *                                                                          *
+ *   FreeCAD is free software: you can redistribute it and/or modify it     *
+ *   under the terms of the GNU Lesser General Public License as            *
+ *   published by the Free Software Foundation, either version 2.1 of the   *
+ *   License, or (at your option) any later version.                        *
+ *                                                                          *
+ *   FreeCAD is distributed in the hope that it will be useful, but         *
+ *   WITHOUT ANY WARRANTY; without even the implied warranty of             *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU       *
+ *   Lesser General Public License for more details.                        *
+ *                                                                          *
+ *   You should have received a copy of the GNU Lesser General Public       *
+ *   License along with FreeCAD. If not, see                                *
+ *   <https://www.gnu.org/licenses/>.                                       *
+ *                                                                          *
+ ***************************************************************************/
+
+#include "PreCompiled.h"
+
+#include "Gizmo.h"
+
+#ifndef _PreComp_
+#include <Inventor/nodes/SoOrthographicCamera.h>
+#include <Inventor/nodes/SoPerspectiveCamera.h>
+#include <Inventor/nodes/SoPickStyle.h>
+#include <algorithm>
+#endif
+
+#include <App/Application.h>
+#include <Base/Converter.h>
+#include <Base/Parameter.h>
+#include <Base/Precision.h>
+#include <Document.h>
+#include <Gui/Inventor/So3DAnnotation.h>
+#include <Gui/Inventor/SoToggleSwitch.h>
+#include <Gui/QuantitySpinBox.h>
+#include <Gui/Utilities.h>
+#include <Gui/View3DInventorViewer.h>
+
+#include "SoLinearDragger.h"
+#include "SoLinearDraggerGeometry.h"
+#include "SoRotationDragger.h"
+#include "SoRotationDraggerGeometry.h"
+
+using namespace Gui;
+
+void Gizmo::setDraggerPlacement(const Base::Vector3d& pos, const Base::Vector3d& dir)
+{
+    setDraggerPlacement(
+        Base::convertTo<SbVec3f>(pos),
+        Base::convertTo<SbVec3f>(dir)
+    );
+}
+
+bool Gizmo::delayedUpdateEnabled()
```

Comment:
typically predicates are started with `is` so it is clear that it is query not action

Reply:
Should be fixed.